bugbug/model.py

```diff
@@ -756,12 +756,3 @@ def items_gen(self, classes):
 class IssueModel(Model):
     def __init__(self, lemmatization=False):
         Model.__init__(self, lemmatization)
-        self.training_dbs = [github.GITHUB_ISSUES_DB]
-
-    def items_gen(self, classes):
-        for issue in github.get_issues():
-            issue_number = issue["number"]
-            if issue_number not in classes:
-                continue
```

Comment:
Moved `self.training_dbs` and `items_gen` to needsdiagnosis model

Reply:
Could we instead keep them here, adding `owner` and `repo` as parameters to `IssueModel` so it can create a `GitHub` instance and store it in `self.github`?

Reply:
Yeah, added it back